### PR TITLE
Add the swagger files

### PR DESCRIPTION
**Pull request by RezaJooyandeh**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by DominikMe — APPROVED** [review]
Lgtm

File: specification/communication/data-plane/readme.go.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  clear-output-folder: true
+```
+
+### Tag: package-2020-04-16-privatepreview and go
+
+These settings apply only when `--tag=package-2020-04-16-privatepreview --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'package-2020-04-16-privatepreview' && $(go)
+namespace: Microsoft.CommunicationServices
```
Comment (ArcturusZhang) [inline specification/communication/data-plane/readme.go.md:17]:
```suggestion
namespace: communication
```

File: specification/communication/data-plane/readme.go.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+output-folder: $(go-sdks-folder)/communicationservices/Generated
```
Comment (ArcturusZhang) [inline specification/communication/data-plane/readme.go.md:18]:
```suggestion
output-folder: $(go-sdk-folder)/services/preview/$(namespace)/2020-04-16-preview/$(namespace)
```

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]
Some comment on the go SDK generation config

File: specification/communication/data-plane/readme.go.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+## Go
```
Comment (ArcturusZhang) [inline specification/communication/data-plane/readme.go.md:1]: If the go SDK generation for data plane is not needed, please just remove this file.

File: specification/communication/data-plane/readme.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+# Code Generation
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+## Go
+
+See configuration in [readme.go.md](./readme.go.md)
```
Comment (ArcturusZhang) [inline specification/communication/data-plane/readme.md:79]: Please remove this if go SDK is not needed

**Review by ArcturusZhang — CHANGES_REQUESTED** [review]

**Review by ArcturusZhang — APPROVED** [review]
Approved since this need no go SDK generation

**Pull request merged**